cypress-tests/cypress/support/pageBuilder/pbUpdatePageBlock.ts

```diff
@@ -0,0 +1,59 @@
+import { gqlClient } from "../utils";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbUpdatePageBlock(id: string, data: any): Promise<any>;
+        }
+    }
+}
+const UPDATE_PAGE_BLOCK = /* GraphQL */ `
+    mutation UpdatePageBlock($id: ID!, $data: PbUpdatePageBlockInput!) {
+        pageBuilder {
+            pageBlock: updatePageBlock(id: $id, data: $data) {
+                data {
+                    id
+                    blockCategory
+                    name
+                    content
+                    createdOn
+                    createdBy {
+                        id
+                        displayName
+                        type
+                    }
+                }
+                error {
+                    code
+                    message
+                    data
+                }
+            }
+        }
+    }
+`;
+
+Cypress.Commands.add("pbUpdatePageBlock", (id, data) => {
```

Comment:
Can we use the new way of writing commands here?

(with the `createGqlQuery` function, like e.g. in `cypress-tests/cypress/support/pageBuilder/pbDeletePage.ts`?)